custom/icds_reports/ucr/data_sources/child_delivery_forms.json

```diff
@@ -229,6 +229,36 @@
         "is_nullable": true,
         "type": "expression",
         "column_id": "sex"
+      },
+        {
+        "display_name": null,
+        "datatype": "string",
+        "expression": {
+          "datatype": null,
+          "type": "property_path",
+          "property_path": ["case_open_child_health_3", "case", "@case_id"]
+        },
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "type": "expression",
+        "column_id": "child_health_case_id"
+      },
+        {
```

Comment:
Its super weird to me why my pyCharms'd default formatting is this one. Do you remember how did we end up having current indentation?

Reply:
I suspect the issue is that pycharm is defaulting to 4 spaces and we use 2 for json in most places (although annoyingly not all). I think its fairly standard in JS to use 2, and I suspect the real reason these files are like that is because it was someone else's editor default.

Reply:
The rest of the expression here consistently uses 2 though, and is properly aligned with expressions above and below it as is the closing brace. The only issue here is the opening braces in both expressions.